src/index.js

```diff
@@ -24,14 +32,12 @@ const pixabayjs = {
   },
 
   imageResultList: function(search, opts, onSuccess, onFailure) {
-    const url = 'https://pixabay.com/api';
-    const conf = this._makeConfig(url, opts);
+    const conf = this._makeConfig('', opts);
```

Comment:
since `_makeConfig` is getting a make-over, would it be cleaner to add `endpoint` as an option?